ivy/functional/frontends/tensorflow/math.py

```diff
@@ -668,3 +668,8 @@ def greater_equal(x, y, name=None):
 def in_top_k(target, pred, k, name=None):
     top_k = ivy.top_k(target, k)
     return ivy.array([val in top_k.values for val in target])
+
+
+@to_ivy_arrays_and_back
+def igamma(a, x, name=None):
+    return ivy.igamma(a, x)
```

Comment:
There is no `igamma` function in the backend which makes this a dummy call 
Kindly either suggest us to add it in the experimental backend using our github template if it is not present in or ToDo list for experimental API